### PR TITLE
new ATAN2 and minor whitespace changes, errata such as duplicate "see also" entries, convert license line from a header to italic text

### PR DESCRIPTION
**Pull request by urbanjost**

### CODE REVIEW

**Comment by henilp105** [comment]
Thanks sir @urbanjost .

**Comment by henilp105** [comment]
#build_preview

File: source/learn/intrinsics/_pages/ACHAR.md [hunk 1]
```diff
@@ -161,13 +161,13 @@ FORTRAN 77 and later, with KIND argument Fortran 2003 and later
 
 ### **See Also**
 
-[**char**(3)](#char),
-[**iachar**(3)](#iachar),
-[**ichar**(3)](#ichar)
+[**char**(3)](CHAR),
+[**iachar**(3)](IACHAR),
+[**ichar**(3)](ICHAR)
```
Comment (awvwgk) [inline source/learn/intrinsics/_pages/ACHAR.md:166]: This change is going to revert the cross-linking introduced with #152

Comment (urbanjost) [reply source/learn/intrinsics/_pages/ACHAR.md:166]: I deleted the old one and made a new pull request. Not sure how the old text got there. Going to delete this and
start over with a new one.  Will just do a single one for the footer line change so only a few files are in the other change

**Pull request closed without merging**